client/layout/sidebar/footer.jsx

```diff
@@ -11,8 +11,8 @@ import Gridicon from 'gridicons';
  */
 import Button from 'components/button';
 import config from 'config';
-import HappychatButton from 'components/happychat/button';
-import { hasActiveHappychatSession } from 'state/happychat/selectors';
+import HappychatButton from 'extensions/happychat/components/button';
```

Comment:
This is a tough one. Again, for this to truly be an extension, there should be no direct reference to it outside of the extension code. However, I haven't seen an extension try to be embedded in another component like this either.

At the least, I think this warrants a TODO comment so we don't forget that this needs to be resolved.